.hooks/pre-commit

```diff
@@ -1,11 +1,10 @@
 #!/bin/sh
-FORBIDDEN=( private_key )
+FORBIDDEN='private_key|sa_key_var|SA_KEY_JSON'
 
-for i in "${FORBIDDEN[@]}"
-do
-  git diff --cached --name-only | \
-        GREP_COLOR='4;5;37;41' xargs grep --color --with-filename -n ${i} --exclude .hooks/pre-commit && \
-        echo 'COMMIT REJECTED: Found "'${i}'" references. Please remove them before committing.' && exit 1
-done
-
-exit 0
+git diff --cached -U0 | grep -E --color ${FORBIDDEN}
```

Comment:
Is there a way to exclude this file? Also, can we list out the files where the keywords appear?

Reply:
We could exclude this file, but changes here are effectively changes to the security policy, so having it warn on that actually seems appropriate.

Listing out the files would require a much more complicated script, which didn't seem worth it. Once you see the strings it should be straightforward to track down where they're located.